src/Rut.php

```diff
@@ -244,7 +244,7 @@ public function calculateVerificationNumber()
         $rut = $this->number;
         $s=1;
         for ($m=0; $rut != 0; $rut /= 10) {
-            $s=($s+$rut % 10 * (9-$m++%6))%11;
+            $s=((int)$s+(int)$rut % (int)10 * ((int)9-(int)$m++%(int)6))%(int)11;
```

Comment:
is it needed to cast integers?
```suggestion
            $s=((int)$s+(int)$rut % 10 * (9-(int)$m++%6))%11;
```

Reply:
Si eso soluciona el problema